main.js

```diff
@@ -692,10 +692,11 @@ addStealExtension(function addMetaDeps(loader) {
 	}
 
 	loader.transpile = function (load) {
+		// TODO this needs to change
```

Comment:
Now that `.transpile()` is called on all formats, prependDeps needs to check the load.format.